src/Support/FieldResolver/DirectiveHandler.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types = 1);
+namespace Rebing\GraphQL\Support\FieldResolver;
+
+use GraphQL\Executor\Values;
+use GraphQL\Language\AST\DirectiveNode;
+use GraphQL\Type\Definition\Directive;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Schema;
+
+class DirectiveHandler
+{
+    /** @var array<\GraphQL\Type\Definition\Directive> */
+    private $directives;
+
+    public function loadDirectivesBySchema(Schema $schema): self
+    {
+        $this->directives = [];
+
+        foreach ($schema->getDirectives() as $directive) {
+            $this->directives[$directive->name] = $directive;
+        }
+
+        return $this;
+    }
+
+    public function getDirective(string $name): ?Directive
+    {
+        return $this->directives[$name] ?? null;
+    }
```

Comment:
With the designed usage you can never run into the situation that the directive isn't present, because the query validation will be executed before. But since the method is public I added it to prevent the "undefined index" exception from being thrown, and anyone can decide how to handle the null value in its application.